code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -179,6 +179,9 @@ Please contact me on #coderbus IRC. ~Carn x
 	//5: Set appearance once
 	appearance = ma_compiled
 
+	//6: Do any species specific layering updates, such as when hiding.
+	update_icon_special()

```

Comment:
Should probably be moved to affect the ma, rather than messing with appearance after (maybe pass it the ma)